src/cosl/coordinated_workers/worker.py

```diff
@@ -255,19 +259,26 @@ def status(self) -> ServiceEndpointStatus:
         # we really don't want this code to raise errors, so we blanket catch all.
         try:
             services = self._container.get_services(*layer.services.keys())
-            running_status = {name: svc.is_running() for name, svc in services.items()}
-            if not all(running_status.values()):
-                if any(running_status.values()):
-                    starting_services = tuple(
+            if services:
+                # avoid vacuous quantification
```

Comment:
Should we bring this comment a few [reading levels](https://hemingwayapp.com/) down? :sweat_smile: 
That being said, the new comment seems to be a pull request comment rather than code comment.

Reply:
it was in fact meant to be a code comment, explaining why we need the `if(services)` check to verify it's not empty before doing an `all` on it, but I'll rephrase